src/assets/db.json

```diff
@@ -346,5 +346,19 @@
    "logo": "https://raw.githubusercontent.com/2ndalpha/gasmask/master/Resources/Images/About%20Box%20Image.png",
    "description": "Gas Mask is a simple hosts file manager for macOS. It allows editing of host files and switching between them.",
    "categories": ["Hosts", "System"]
+  },
+  {
+    "id": 50,
+    "name": "obsidian",
+    "logo": "https://avatars.githubusercontent.com/u/65011256?s=200&v=4",
+    "description": "Obsidian is a powerful and extensible knowledge base that works on top of your local folder of plain text files. ",
+    "categories": ["Productivity", "Note Taking"]
+  },
+  {
+    "id": 51,
+    "name": "Notion",
+    "logo": "https://upload.wikimedia.org/wikipedia/commons/4/45/Notion_app_logo.png",
+    "description": "One workspace. Every team. We’re more than a doc. Or a table. Customize Notion to work the way you do.",
+    "categories": ["Productivity", "Note Taking"]
   }
```

Comment:
Since the db is updated, we can update the new id to 51 and 52. Notion is used as `notion` in brew cask. You can lower the initials. There seems to be extra space at the end of the Obsidian description. Apart from that, thanks for 2 new apps. 💫 @volkanto